assembly/src/assembler/mast_forest_builder.rs

```diff
@@ -16,7 +16,7 @@ use vm_core::{
 };
```

Comment:
Needed for below suggestion
```suggestion
    utils::collections::KvMap,
};
```

Reply:
done